.github/workflows/_test-tf-dflook-cleanup.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/_test-tf-dflook-cleanup.yaml'
+      - '.github/workflows/tf-dflook-cleanup.yaml'
+      - 'tests/terraform/**'
+  push:
+    branches:
+      - main
+
+jobs:
+  # Hack to wait for the feature workflow to finish
+  sleep:
```

Comment:
Why not only 1 _test-tf-dflfock-worflow.yaml with the 2/3 jobs:
- test_tf_dflook_apply
- test_tf_dflook_cleanup
- test_tf_dflook_feature
Is there a side effect because it will run in the same action ?

Reply:
I only want to run the workflows if there is changes in the PR